web/modules/custom/dbcdk_community/src/Form/QuarantineEditForm.php

```diff
@@ -0,0 +1,203 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\dbcdk_community\Form\QuarantineEditForm.
```

Comment:
There seems to be a lot of duplication here between the create and edit form. How about reducing this to a single form based on the following principles:

1. `create` fetches a `Quarantine` from the API if a request argument is present or creates an empty one if not
2. The empty or populated `Quarantine` is passed to the form constructor
3. `buildForm` creates the form with default values from the `Quarantine` object and [stores the `Quarantine` in the form state](https://github.com/DBCDK/biblo-admin/pull/39/files#diff-43b66a6be6f52cc9f63ae814391bcc2eR82).
4. [`submitForm` retrieves the `Quarantine` from the form state](https://github.com/DBCDK/biblo-admin/pull/39/files#diff-43b66a6be6f52cc9f63ae814391bcc2eR98), sets the values and calls create or upsert on the API based on whether the `Quarantine` has an id or not.